pkg/local_workflows/report_analytics_workflow.go

```diff
@@ -132,8 +132,7 @@ func reportAnalyticsEntrypoint(invocationCtx workflow.InvocationContext, inputDa
 		// send to V2 analytics endpoint
 		callErr := callEndpoint(invocationCtx, input, url)
 		if callErr != nil {
-			err := fmt.Errorf("error calling endpoint for input at index %d: %w", i, callErr)
-			return nil, err
+			logger.Printf("error calling endpoint for input at index %d: %v", i, callErr)
```

Comment:
Suggestion: The decision what to do in case of this error should be on the application not on the workflow, I would recommend to do the change in snyk-ls and leave this code unchanged.